apicompat/baselines/ApiCompatBaseline.portable4.0-Profile1.txt

```diff
@@ -2,15 +2,8 @@ DEFAULT_APPNAME Warning: 0 : Found 'mscorlib' with PublicKeyToken 'b77a5c561934e
 DEFAULT_APPNAME Warning: 0 : Found 'System.Core' with PublicKeyToken 'b77a5c561934e089' instead of '7cec85d7bea7798e'.
 DEFAULT_APPNAME Warning: 0 : Found 'System' with PublicKeyToken 'b77a5c561934e089' instead of '7cec85d7bea7798e'.
 DEFAULT_APPNAME Warning: 0 : Found 'System.Xml' with PublicKeyToken 'b77a5c561934e089' instead of '7cec85d7bea7798e'.
-DEFAULT_APPNAME Error: 0 : Failed to find or load matching assembly 'System.Xml.Serialization'.
+DEFAULT_APPNAME Warning: 0 : Found 'System.Xml.Serialization' with PublicKeyToken 'b77a5c561934e089' instead of '31bf3856ad364e35'.
+DEFAULT_APPNAME Error: 0 : Unable to resolve assembly 'Assembly(Name=System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)' referenced by the implementation assembly 'Microsoft.Cci.DummyModule'.
 Compat issues with assembly System.Xml.Serialization:
 TypesMustExist : Type 'System.ServiceModel.XmlSerializerFormatAttribute' does not exist in the implementation but it does exist in the contract.
```

Comment:
@terrajobst what do you think about this one? This lives in System.ServiceModel.dll on .NET but is part of the PCL profiles. The usage data isn't very high ~2.5% at the most so I'm inclined to keep it out for now.